src/deployment/deploy.py

```diff
@@ -1236,8 +1249,8 @@ def main() -> None:
     formatter = argparse.ArgumentDefaultsHelpFormatter
     parser = argparse.ArgumentParser(formatter_class=formatter)
     parser.add_argument("location")
-    parser.add_argument("resource_group")
-    parser.add_argument("application_name")
+    parser.add_argument("resource_group", type=lower_case)
```

Comment:
the resource group does not need the restriction only the application_name
```suggestion 
    parser.add_argument("resource_group")
```